### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Splend1ed**

### CODE REVIEW

File: app/managers.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self):
+        self.connection = sqlite3.connect("cinema.db")
+        self.table_name = "actor"
```
Comment (Abnormaltype) [inline app/managers.py:9]: Does this DB contain only one author?

**Review by Abnormaltype — CHANGES_REQUESTED** [review]

File: app/managers.py (hunk 1 above)
Comment (Splend1ed) [reply app/managers.py:9]: What u mean?

Comment (Abnormaltype) [reply app/managers.py:9]: DB has the name `actor` which means, DB contains only one author

**Review by Abnormaltype — APPROVED** [review]
well done!

**Pull request closed without merging**